packages/logger/README.md

```diff
@@ -489,22 +489,42 @@ Different types of data are serialized differently before being output as JSON:
       // }
       ```
 
-      Errors found in sub-properties of the log data are _not_ serialized like this. This is for performance reasons: looping over every nested property to check if it's an error is expensive. Do **not** do this:
+    Errors found in sub-properties of the log data are _not_ serialized like this due to performance reasons: looping over every nested property to check if it's an error is expensive (do **not** do this). Also, be mindful of passing in errors with a custom error property e.g. `myErrorProperty`, as you'll have to serialize them yourself like below:
 
-      ```js
-      logger.info({ err: new Error('Oops') });
-      // Outputs:
-      // {
-      //     "level": "info",
-      //     "err": {}
-      // }
-      ```
+    ```js
+    logger.info({ myErrorProperty: serializeError(new Error('Oops')) });
+    ```
 
-      If you _need_ to pass error objects as a property, you must serialize it yourself:
+    ... if not, you'll get the following output:
+
+    ```js
+    logger.info({ myErrorProperty: new Error('Oops') });
+    // Outputs:
+    // {
+    //     "level": "info",
+    //     "myErrorProperty": {}
+    // }
+    ```
+    
+    The exception to this is if the sub-property name is either `error` or `err`, as these are automatically serialized. E.g.
 
       ```js
-      logger.info({ err: serializeError(new Error('Oops')) });
-      ```
+    logger.info({ error: new Error('Oops') });
+    // Outputs:
+    // {
+    //     "level": "info",
+    //     "error": {
+    //         "cause": null,
```

Comment:
This line is perfectly aligned on my local IDE, but when I push my changes to GitHub it's then mis-aligned 😭